.github/workflows/cd.yml

```diff
@@ -0,0 +1,51 @@
+name: cd
+
+on:
+  push:
+    tags:
+      - "[v]?[0-9]+.[0-9]+.[0-9]+*"
+
+env:
+  CONTAINER_REGISTRY: ghcr.io
+
+jobs:
+  build_and_publish_image:
```

Comment:
I was actually thinking to make this generic so that this same job can be used for different docker build and push — since the first few steps of waiting for CI to pass and logging into GCR do not need to be repeated for different docker images in this same repo

Reply:
^ignore this, I realised the `waiting for CI to pass` step needs to be specific for notary docker image in this case, because the `wait on check action` can only check for a specific package (i.e. notary-server) in the [matrix](https://github.com/tlsnotary/tlsn/blob/1c7273143ed8e2b8d05a5bc53e630b98f64badb4/.github/workflows/rust.yml#L19-L30) we used in `ci.yml`, instead of the overall test result of all the packages, more info [here](https://github.com/lewagon/wait-on-check-action#check-name) under the example where they use strategy matrix